Session01/htmlcss2/account.html

```diff
@@ -0,0 +1,97 @@
+            </div>
+        </article>
+    </section>
+
+    <section>
+        <section class="details-container">
+            <div class="date-container">
+                <button type="button"><b>&lt;</b></button>
+                <span id="currentDate">10/13</span>
+                <button type="button"><b>&gt;</b></button>
+            </div>
+            <div class="checkbox-container">
+                <label class="checkbox-in">
+                    <input type="checkbox"checked />수입</label>
+                <label class="checkbox-out">
+                    <input type="checkbox"checked />지출</label>
+            </div>
+        </header>
+        <div class="borderline"></div>
+
+        <ul class="inoutlist">
+            <li>
+                <span class="list-category">식비</span>
+                <span class="list-description">스타벅스</span>
+                <span class="list-amount-minus">- 10,000원</span>
+                <button type="button">X</button>
+            </li>
+            <li>
+                <span class="list-category">용돈</span>
+                <span class="list-description">고모부</span>
+                <span class="list-amount-plus">+ 250,000원</span>
+                <button type="button">X</button>
+            </li>
+            <li>
+            <span class="list-category">식비</span>
+            <span class="list-description">스타벅스</span>
+            <span class="list-amount-minus">- 5,000원</span>
+            <button type="button">X</button>
+            </li>
+            <li>
+                <span class="list-category">서적</span>
+                <span class="lis-description">자구교재</span>
+                <span class="list-amount-minus">- 50,000원</span>
+                <button type="button">X</button>
+            </li>
+            <li>
+                <span class="list-category">식비</span>
+                <span class="list-description">냉우동</span>
+                <span class="list-amount-minus">- 18,000원</span>
+                <button type="button">X</button>
+            </li>
+            <li>
+            <span class="list-category">쇼핑</span>
+            <span class="list-description">머리띠</span>
+            <span class="list-amount-minus">- 4,900원</span>
+            <button type="button">X</button>
+            </li>
+            <li>
+                <span class="list-category">식비</span>
+                <span class="list-description">초코칩</span>
+                <span class="list-amount-minus">- 1,500원</span>
+                <button type="button">X</button>
+            </li>
+            <li>
+                <span class="list-category">학용품</span>
+                <span class="list-description">화이트</span>
+                <span class="list-amount-minus">- 1,200원</span>
+                <button type="button">X</button>
+            </li>
+        </ul>
+    </section>
+    <footer>
+        <button type="button">
+            <button id="add">추가</div>
+        </button>
```

Comment:
```suggestion
        <button type="button">추가
        </button>
```
버튼이 중첩된건 어색한 구조에요! 이렇게 줘야하고, 그 안에서 뭔가 한번더 감싸야 하는 스타일링이 필요하다면 span 태그를 이용해봅시당